web/styles/dark_theme.css

```diff
@@ -124,6 +118,19 @@
         color: inherit;
     }
 
+    input:not(.input-element) {
```

Comment:
I wonder if it's worth clearing these styles out of `dark_theme.css`...

Reply:
Good thought. Can you track this, or should we open up an issue?

Reply:
Tracked in #35135.